lib/mayaUsd/CMakeLists.txt

```diff
@@ -163,14 +163,14 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/lib/usd/plugInfo.json
 # -----------------------------------------------------------------------------
 # subdirectories
 # -----------------------------------------------------------------------------
-add_subdirectory(ae)
```

Comment:
After discussion with Krystian, we decided to move this "ae" folder underneath a new "resources" folder. So below you'll see all the ae template (py) files being moved.